modules/virt-installing-qemu-guest-agent-on-linux-vm.adoc

```diff
@@ -7,14 +7,13 @@
 [id="virt-installing-qemu-guest-agent-on-linux-vm_{context}"]
 = Installing the QEMU guest agent on a Linux VM
 
-The `qemu-guest-agent` is widely available and available by default in {op-system-base-full} virtual machines (VMs). Install the agent and start the service.
+The `qemu-guest-agent` is available by default in {op-system-base-full} virtual machines (VMs)
```

Comment:
```suggestion
The `qemu-guest-agent` agent is available by default in {op-system-base-full} virtual machines (VMs).
```

- Missing full stop.
- Object (`qemu-guest-agent`) name is missing 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

> An object reference is when you are referring to the actual instance of an API object, where the object name is important.